extensions/binding/org.eclipse.smarthome.binding.ble/src/main/java/org/eclipse/smarthome/binding/ble/BleBridgeApi.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.ble;
+
+/**
+ * The {@link BleBridgeApi} class defines the standard bridge API that must be implemented by Thing implemented in the
+ * BLE host.
+ * <p>
+ * <b>Scanning</b>
+ * The API assumes that the bridge is "always" scanning to enable beacons to be received.
+ * The bridge must decide to enable and disable scanning as it needs. This design choice avoids interaction between
+ * higher layers where a binding may want to enable scanning while another needs to disable scanning for a specific
+ * function (e.g. to connect to a device). The bridge should disable scanning only for the period that is needed.
+ *
+ * @author Chris Jackson - Initial contribution
+ */
+public interface BleBridgeApi {
+
+    /**
+     * Starts an active scan on the Bluetooth interface.
+     *
+     * @return true if the scan was started
+     */
+    boolean scanStart();
```

Comment:
It is not clear for me, how I am suppose to take the scan results. I expected that I could register some callback that will be called in the implementation when a scan result is found. 

At the moment the BridgeHandler calls a method in the DiscoveryService, but what should we do if we want to decouple them and use only the BleBridgeApi interface ?